lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -110,6 +112,36 @@ def resolve_database_info_from_model(model_name)
         { error: e.full_message(highlight: false) }
       end
 
+      def resolve_association_target(params)
+        const = ActiveSupport::Inflector.safe_constantize(params[:model_name])
+        unless active_record_model?(const)
+          return {
+            result: nil,
+          }
+        end
+
+        association_klass = case params[:association_type].intern
+        when :has_many
+          ActiveRecord::Associations::Builder::HasMany.build(const, params[:association_name].intern, nil, {}).klass
```

Comment:
Someone may have an invalid association specified, so we'll need handle that, e.g. by rescuing `NameError`.